lisp/ess-r-completion.el

```diff
@@ -367,6 +367,28 @@ To be used instead of ESS' completion engine for R versions >= 2.7.0."
 ;;; COMPANY
 ;;; http://company-mode.github.io/
 
+(defvar company-R-library--cache ())
+(defun company-R-library-all-completions ()
+  (or company-R-library--cache
```

Comment:
We don't need this one on emacs side if `installed.packages` uses internal cache. Have you checked that out?

Reply:
only as far as reading its documentation. I figured it would still be faster to cache in lisp since its tiny memory, but could remove if you want.

Reply:
The issue is that package list can change over time and different processes could end up having different lists (think of remotes). Even on the same machine different R processes could end up with different package list for a number of reasons.

Reply:
good point, I will remove the cache